gpcontrib/arenadata_toolkit/.gitignore

```diff
@@ -1,4 +1,6 @@
 # Generated subdirectories and files
+/expected/tablespace_location.out
```

Comment:
?

Reply:
This file is generated from new file: "output/tablespace_location.out"